src/objects/Mesh.js

```diff
@@ -49,6 +49,59 @@ class Mesh extends Object3D {
 
 	}
 
+	volume( precision ) {
+
+		function calculateVolume( { geometry, precision } ) {
+
+			let volume = 0;
+			const vertices = geometry.attributes.position.array;
+			let indexes = geometry.index ? geometry.index.array : null;
+
+			function triangularVolume( p1, p2, p3 ) {
+
+				const v321 = p3.x * p2.y * p1.z;
```

Comment:
I assume this implementation is based on some kind of resource. Do you mind adding it as a comment?

Reply:
Ok, I see the method here – and tetrahedral triangulation can be skipped. I might suggest adding a comment on this method to explain that it is computing the _signed volume_ of a tetrahedron, having the triangle as one face and the origin as the other.